### ISSUE TITLE
Sticky notifications not working

### ISSUE THREAD

**Issue by brunogoossens** [timche/gmail-desktop#15]
After you change the notification style for the Gmail desktop app to "Alert" (=sticky notification). When you receive a new mail, the notification disappears after a few seconds. I like it to be sticky.

**Comment by timche** [comment]
Will consider this make it configurable.

**Comment by markypython** [comment]
So, I circled back around to this one and did a bunch more testing, trying different things, and what I found was Gmail's code does some crazy magic deep down inside that is preventing me from overriding the window.Notification class.  I was able to get it to almost work where everything was firing using the electron notifications but the hangup was that the callback function for onclick (set by Gmail using `addEventListener`) was not working like it was with the native notifications.

All that to say, I think we have three options:

1. Leave it as is and don't support sticky notifications
1. Change the notifications to support sticky notifications and not support opening the message on click
1. Listen to the network traffic for the new email request so we know the email identifier and can open the email without using the onclick listener provided by the Gmail notification.  Haven't tested this at all, but it would probably work something like this library (just much simpler) https://github.com/KartikTalwar/gmail.js/blob/9e69719b294c7386017b41bb37539c9588e4701f/src/gmail.js#L1164

@timche Thoughts?
